ldm/invoke/args.py

```diff
@@ -866,6 +866,11 @@ def _create_dream_cmd_parser(self):
             default=32,
             help='When outpainting, the tile size to use for filling outpaint areas',
         )
+        postprocessing_group.add_argument(
+            '--new_prompt',
```

Comment:
Because `prompt` is used for the original prompt used to generate the image (if present), and reusing the same name would cause confusion within the code.